app/views/adventure_groups/_adventure_group.erb

```diff
@@ -0,0 +1,16 @@
+<% ratio = adventure_group.points.fdiv(max) %>
+<div class="col-xs-3 block">
+  <div class="text"><%= adventure_group.group.name %></div>
+  <% if ratio > 0.75 %>
+    <img class="part" src="/assets/round-table/mission4.svg"></img>
```

Comment:
> I think this would be good with image_tags to be able to use the assets when precompiled.
> Or any reason to use hard-coded paths?
> 
> image_path('round-table/missionx.svg', class: 'part')( I think )

No reason. I was just hinking in HTML :wink:
Do you mean `image_tag`?